pytorch/Dockerfile

```diff
@@ -109,6 +109,57 @@ RUN wget -q  --no-check-certificate https://raw.githubusercontent.com/oneapi-src
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/docker/third-party-programs-pytorch.txt -O /licensing/third-party-programs-pytorch.txt && \
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/LICENSE -O /licensing/LICENSE
 
+FROM multinode AS hf-genai
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends --fix-missing \
+    google-perftools \
+    libjemalloc2 \
+    libomp-dev \
+    numactl \
+    git
+
+COPY hf-genai-requirements.txt .
+
+RUN python -m pip install --no-cache-dir -r hf-genai-requirements.txt
+
+# Install transformers from source to provide support for transformers example scripts
+RUN git clone https://github.com/huggingface/transformers.git && \
```

Comment:
For `git clone` within a container, I'd always make sure I have:
```
ARG BRANCH_NAME=<MY_BRANCH_NAME>
RUN git clone --single-branch --branch=${MY_BRANCH_NAME} REPO_URL
```

For the `branch` in general, I'd use the latest tag or a specific commit ID that you know it works and it's CVE free.

Lastly, can we install `transformers` from `pip` rather than source and then only install the remaining ones from the `requirements.txt` file?

Reply:
Let me try and see if that works. Thanks for the suggestion

Reply:
I ended up removing it totally from the Dockerfile and added into hf-genai-requirements.txt